.github/CODEOWNERS

```diff
@@ -1,5 +1,26 @@
 # Code owners file.
 # This file controls who is tagged for review for any given pull request.
+#
+# What is a "CODEOWNER"?
+#
+# A CODEOWNER MUST ____ for packages they own:
+# - introduce themselves at a Python SIG meeting
+# - demonstrate advanced knowledge of the dependencies
+# - respond to issues when they are notified
+# - fix failing unit tests or any other blockers to the CI/CD workflow
+# - update usage of `opentelemetry-python-core` APIs upon the introduction of breaking changes
+#
+# A CODEOWNER SHOULD:
+# - be a member of the OpenTelemetry community
+#
```

Comment:
```suggestion
```

As defined [here](https://github.com/open-telemetry/community/blob/main/community-membership.md) to be a member of the community someone must have been sponsored by 2 approvers or maintainers and have multiple contributions to the project. I think that is too much to ask for contrib codeowners because it seems to me that it is very possible that someone may only be interested in creating or maintaining a contrib package because that person has an active interest in a particular library, but that same person does not have any other interest in the OpenTelemetry project. I am perfectly fine with having a contrib package codeowner who just works on that certain contrib package.

Reply:
I agree, that's why I only put that as a "SHOULD" in order to encourage the developer to get involved in the OTel community. I think it would beneficial to expose these contributors to OTel as a whole because hopefully they will feel inclined to speak up when they notice something that goes against the patterns they introduce for packages here.

I think it's okay to leave this since it is only "SHOULD" not "MUST".